src/validation/loader.rb

```diff
@@ -71,12 +71,33 @@ def answer(absolute_path, section)
         validator.data
       end
 
+      # XXX Extract object for loading configure data?
       def configure_data_tree
```

Comment:
This can probably be renamed to `configure_tree`. The `data` is superfluous at this point.

Reply:
Good point - though actually looking at this again the existence of this method seems a bit pointless, as it's just called from `configure_data` and is the only thing that does. Therefore I've inlined the whole thing in https://github.com/alces-software/metalware/commit/2df9b97764c32129088b64a3c348a6afb1a5ce1a.